### PR TITLE
Fix two left-over path util usages

### PR DESCRIPTION
**Pull request by leofeyer**
@m-vo Not sure if you left these unchanged for a reason?

### CODE REVIEW

File: core-bundle/src/Cache/ContaoCacheWarmer.php [hunk 1]
```diff
@@ -162,7 +162,7 @@ private function generateLanguageCache(string $cacheDir): void
                 $subfiles = $this->finder
                     ->findIn(Path::join('languages', $language))
                     ->files()
-                    ->name('/^'.$name.'\.(php|xlf)$/')
+                    ->name("/^$name\\.(php|xlf)$/")
```
Comment (m-vo) [inline core-bundle/src/Cache/ContaoCacheWarmer.php:165]: Why did you add another `\`? Wouldn't that actually match a backslash now?

**Comment by m-vo** [comment]
Not sure how I could miss the one in the `CrawlCommand` :smile:.  The other one was intentional, but thinking about it again it makes much more sense to replace it as well. So :+1: .

File: core-bundle/src/Cache/ContaoCacheWarmer.php (hunk 1 above)
Comment (leofeyer) [reply core-bundle/src/Cache/ContaoCacheWarmer.php:165]: It wasn't me. The `EscapeImplicitBackslashesFixer` wanted it this way. 😄

Comment (leofeyer) [reply core-bundle/src/Cache/ContaoCacheWarmer.php:165]: Isn't this the typical "why you need `\\\\` within double quotes to actually have one literal backslash" case?

Comment (m-vo) [reply core-bundle/src/Cache/ContaoCacheWarmer.php:165]: Yeah, I guess you're right.

**Review by m-vo — APPROVED** [review]

File: core-bundle/src/Cache/ContaoCacheWarmer.php (hunk 1 above)
Comment (m-vo) [reply core-bundle/src/Cache/ContaoCacheWarmer.php:165]: Although, to be honest - I do not fully understand why this is the case...

When I try it with `preg_match` it's the same outcome, but when I try it e.g. on https://regex101.com `\\.` actually matches a literal `\` + any character while `\.` matches the literal `.` (which is what it should be).

Comment (leofeyer) [reply core-bundle/src/Cache/ContaoCacheWarmer.php:165]: https://3v4l.org/h2Og9

Comment (ausi) [reply core-bundle/src/Cache/ContaoCacheWarmer.php:165]: @m-vo `"\\."` is a php string, not a RegExp directly. 
So the PHP string `"\\."` results in the RegExp `\.` which results in the literal string `.` to match.
